aries/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+
+name = "aries"
+version = "0.1.0"
+authors = ["Absa Group Limited", "Hyperledger Indy Contributors <hyperledger-indy@lists.hyperledger.org>"]
+publish = false
+license = "Apache-2.0"
+edition = "2018"
+
+[lib]
+name = "aries"
+path = "src/lib.rs"
+crate-type = ["staticlib","rlib", "cdylib"]
```

Comment:
I suppose these could be removed, as it would no longer be compiled as static/dynamic library

Reply:
Correct